permission_handler/pubspec.yaml

```
@@ -1,6 +1,6 @@
 name: permission_handler
 description: Permission plugin for Flutter. This plugin provides a cross-platform (iOS, Android) API to request and check permissions.
-version: 5.0.1+1
+version: 6.0.0-nullsafety.0
```

Please remove the trailing `.0` as we are following the pattern of not adding a `.0` for the first `nullsafety` release.
```suggestion
version: 6.0.0-nullsafety
```